internal/code-generator/generator/generator_test.go

```diff
@@ -244,8 +241,29 @@ func (s *GeneratorTestSuite) TestGenerationFiles() {
 			wantOut:     mimeTypeGold,
 		},
 	}
+}
 
-	for _, test := range tests {
+func (s *GeneratorTestSuite) TearDownSuite() {
+	if s.cloned {
+		err := os.RemoveAll(s.tmpLinguist)
+		assert.NoError(s.T(), err)
```

Comment:
Do we want to fail the test if the cleanup fails? That isn't really actionable for someone debugging the unit.